### ISSUE TITLE
debugadapter error when remote call itself errors

### ISSUE THREAD

**Issue by JanSharp** [justarandomgeek/vscode-factoriomod-debug#30]
I'm not sure if the title is perfectly accurate, but a remote call that crosses the mod border that's given invalid aruments such as functions causes this:
```
__debugadapter__/debugadapter.lua:187: attempt to index local 'lastframe' (a nil value)
```
however looking at the source it seems very context specific so here are 2 example mods to fully reproduce it:
[DemoMods.zip](https://github.com/justarandomgeek/vscode-factoriomod-debug/files/5742270/DemoMods.zip)

**Comment by tburrows13** [comment]
I get the same error when I do `settings.global["spidertron-enhancements-enter-entity-base-game"] = {value = "yes"}` in `script.on_init`.

The mistake is because `value` should be a boolean.

**Comment by tburrows13** [comment]
Same error again, but this time when calling `surface.create_entity` with an incorrect `name` argument.

**Comment by justarandomgeek** [comment]
This should be resolved in 1.1.3
